javascript/src/main/pages/Admin/SearchedTerms.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+
+const SearchedTerms = () => {
+    
+    console.log(SearchedTerms);
+    const columns = [{
+        dataField: 'term',
+        text: 'Term',
+        sort:true
+    }, {
+        dataField: 'id',
+        text: 'ID',
+        sort:true
+    }, {
+        dataField: 'count',
+        text: 'Count',
+        sort:true
+    }];
+
+    return (
+        <>
+            <p>
+                <font size = "5">
+                    <b>Searched Terms</b>
+                </font>
+            </p>
```

Comment:
Instead of: 

```
 <p>
                <font size = "5">
                    <b>Searched Terms</b>
                </font>
            </p>
```

Maybe consider:
```
                    <h2>Searched Terms</h2>
```